### PR TITLE
installing and deploying cloudy in several instances

### PR DESCRIPTION
**Pull request by ismferd**
Hi again,

in order to install cloudy in several instances, I want to add an Ansible script to do that.

### CODE REVIEW

**Comment by ismferd** [comment]
@rogerpueyo ☝️ , when you can of course

File: README.md [hunk 1]
```diff
@@ -19,3 +19,13 @@ and let the magic begin! After the process has finished, you can browse the Clou
 
 ### Log
 The output of the whole Cloudynitzar process is logged to `/var/log/cloudy/cloudynitzar.log`.
+```
+
+If you want to "cloudinizer" a few machines then you should add hostnames in host file and launch:
```
Comment (rogerpueyo) [inline README.md]: Can you please better explain this step?

File: README.md [hunk 2]
```diff
@@ -19,3 +19,13 @@ and let the magic begin! After the process has finished, you can browse the Clou
...
+
+**NOTE:** For go ahead with this step you will need [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
```
Comment (rogerpueyo) [inline README.md]: To proceed with/Before going ahead with/To perform

**Review by rogerpueyo — CHANGES_REQUESTED** [review]
Please check the comments,

Thanks,
Roger

**Comment by ismferd** [comment]
Sure, 
Can you check it again?  thanks @rogerpueyo

**Comment by ismferd** [comment]
@rogerpueyo , I checked your comments and I added info

**Pull request merged**